extensions/wikia/PhalanxII/hooks/PhalanxUserBlock.class.php

```diff
@@ -26,7 +26,11 @@ function __construct(){
 	 *
 	 * @return bool
 	 */
-	static public function blockCheck( User $user, $shouldLogBlockInStats = true ) {
+	static public function blockCheck( User $user, $shouldLogBlockInStats = true, $global = true ) {
```

Comment:
:speech_balloon: — for backwards compatibility, assume we are dealing with a global block; do nothing in case of a local block.